integration-tests/src/test/scala/org/constellation/testutils/MyMatchers.scala

```diff
@@ -0,0 +1,17 @@
+package org.constellation.testutils
+
+import org.scalatest.enablers.Emptiness
+import org.scalatest.matchers.dsl.MatcherFactory1
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.matchers.{MatchResult, Matcher}
+
+trait CustomMatchers extends Matchers {
+
+  val containTheSameElements: MatcherFactory1[Seq[_], Emptiness] = (not be empty).and(Matcher { (left: Seq[_]) =>
+    MatchResult(
+      left.forall(_ == left.head),
```

Comment:
maybe just `left.toSet.size == 1`?

Reply:
@marcinwadon it is elegant but less efficient than `forall`:

It won't terminate early as `forall` stops once it finds first element which does not match the predicate.
If you instantiate `Set` with `"A", "B", "C", "C, "C"` then `Set` makes it step by step. At some step is has let's say `Set("A", "B")` (and now it tries to add `C` here). For our case it is enough to say that it does not match the result and we could stop there but `Set` will then compare every following element with both "A" and "B" so all the "C" will be compared with "A" and "B".

Been there already ;d